lib/beaker/ssh_connection.rb

```diff
@@ -319,6 +325,12 @@ def scp_from source, target, options = {}
       result.stdout = "\n"
 
       begin
+        # This is probably windows with an environment variable so we need to
+        # expand it.
+        source = self.execute(%{echo "#{source}"}).output.strip.gsub('"','') if source.include?('%')
+
+        require 'pry'
+        binding.pry
```

Comment:
Please remove this, the CI got stuck

Reply:
@gimmyxd Sorry about that :-|. Thought that I had caught them all.